sway/criteria.c

```diff
@@ -144,6 +144,35 @@ static void find_urgent_iterator(struct sway_container *con, void *data) {
 	list_add(urgent_views, con->view);
 }
 
+bool has_container_criteria(struct criteria *criteria) {
```

Comment:
Since this isn't used outside of the file and is not included in the header, can you please make this `static`?